src/bidiServerRunner.ts

```diff
@@ -27,19 +27,31 @@ export class BidiServerRunner {
   // Once run, waits for connection. For each connection, calls `onOpen` and stores the returned `stateObject`.
   // `stateObject` passed to `onClose` afterwards.
   static run(
+    bidiPort: number,
     onNewBidiConnectionOpen: (bidiServer: IServer) => Promise<BrowserProcess>,
     onBidiConnectionClosed: (browserProcess: BrowserProcess) => void
   ) {
     const self = this;
-    const bidiPort = parseInt(process.env.PORT) || 8080;
 
     const server = http.createServer(function (request, response) {
       debugInternal(new Date() + ' Received request for ' + request.url);
-      response.writeHead(404);
+
+      // Needed for WPT compatibility.
+      response.writeHead(200, { 'Content-Type': 'text/plain' });
```

Comment:
```suggestion
      response.writeHead(200, { 'Content-Type': 'application/json; charset=utf-8', 'Cache-Control': 'no-cache' });
```

Matching https://w3c.github.io/webdriver/#dfn-send-a-response. Maybe link that here too.